app/assets/javascripts/covid19/src/data/Covid19InfoFromJsonRepository.ts

```diff
@@ -217,6 +215,36 @@ function getDetails(pdb: Data.Pdb): Maybe<Details> {
     };
 }
 
+function getPdbValidations(pdb: Data.Pdb): PdbValidation[] {
+    return pdb.refModels
+        ? pdb.refModels?.map(refModel => {
```

Comment:
Since the block is returning structural data (objects without type), we should inform TS what it should expect to be returned. To to this: `pdb.refModels?.map((refModel): PdbValidation => {`